File: examples/petstore/oseg_generated/php/GetInventory_default_example.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPI\Client\Configuration::getDefaultConfiguration();

try {
    $response = (new OpenAPI\Client\Api\StoreApi($config))->getInventory();

    print_r($response);
} catch (OpenAPI\Client\ApiException $e) {
    echo 'Exception when calling Store#getInventory: '
        . print_r($e->getResponseObject());
}

File: examples/petstore/oseg_generated/csharp/GetInventory_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPITools.Api;
using Org.OpenAPITools.Client;
using Org.OpenAPITools.Model;

public class GetInventoryDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            var response = new StoreApi(config).GetInventory();

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Store#GetInventory: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
